app/javascript/chat/ChatChannelSettings/ChatChannelSettings.jsx

```diff
@@ -149,6 +157,7 @@ export default class ChatChannelSettings extends Component {
 
   updateMemberships = (membershipId, response, membershipStatus) => {
     if (response.success) {
+      this.componentDidMount();
```

Comment:
Avoid calling component lifecycle methods explicitly. If there is common functionality that can be reused, wrap it in a function/method that can be used here as well as in `componentDidMount()`.